src/plugins/sync-plugin.js

```diff
@@ -469,8 +469,7 @@ const createTextNodesFromYText = (text, schema, mapping, snapshot, prevSnapshot,
   const nodes = []
   const deltas = text.toDelta(snapshot, prevSnapshot, computeYChange)
   try {
-    for (let i = 0; i < deltas.length; i++) {
-      const delta = deltas[i]
+    for (const delta in deltas) {
```

Comment:
Please don't use for-in loops for arrays as they fail to transpile in old typescript versions. I deliberately only use normal for loops.